### DISCUSSION TITLE
Blogpost draft: glTF gravity spec

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/media#96] [Ideas]
OMI Physics Gravity. Physics gravity allows you to specify custom gravity directions and shapes inside of your GLTF scene. You can kind of think of it like allowing Super Mario Galaxy-style gameplay. 
So, imagine taking a Super Mario Galaxy map and putting it in a glTF file.

Trying to read the transcript OR listen to the audio of the meeting on 17 april where Aaron spoke through this, but file stops around 20 minutes? When I open the audio file it shows 54min but then quickly contracts to 20min. @madjin can you take a look at it please?

embed the video: https://omigroup.org/wp-content/uploads/2024/06/Screencast-from-2024-04-17-16-27-40.webm

**Comment by jimmy6DOF** [comment]
I think we are going to need a re-play of that session the audio file is corrupt for me too and recoding with ffmpeg didn't help oh well back to the drawing board.

also a good place to keep an eye on here is Jin's [OMI Archive](https://github.com/omigroup/omi-archive)
